<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageAnalysisDialog</class>
 <widget class="QWidget" name="ImageAnalysisDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="ImageWidget" name="imageWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="loadButton">
         <property name="text">
          <string>Load image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadROIButton">
         <property name="text">
          <string>Load ROI</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="computeHistogramButton">
         <property name="text">
          <string>Compute histogram</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveHistogramButton">
         <property name="text">
          <string>Save histogram</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="calculateSkinRegionButton">
         <property name="text">
          <string>Show skin color pixels</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showCurrentHistogramButton">
         <property name="text">
          <string>Show current histogram</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showSaveHistogramButton">
         <property name="text">
          <string>Show saved histogram</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="thresholdLabel">
           <property name="text">
            <string>Threshold</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="thresholdSpinBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QWidget</extends>
   <header>C:\Projekte\C++\Kinect_Tracker\KinectTracker\inc\ImageWidget\inc\ImageWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
